cmake/generic.cmake

```diff
@@ -483,6 +483,19 @@ function(py_test TARGET_NAME)
   endif()
 endfunction()
 
+function(py_test_modules TARGET_NAME)
```

Comment:
I like `py_test_modules` and it has the potential in many other projects.  For the near future, is it the only purpose of `py_test_modules` to bundle operators unit tests? If so, how about we move it to `fluid/tests/unittests/CMakeLists.txt`?

Reply:
Done